src/integration/configuration.ts

```
@@ -0,0 +1,180 @@
+                const server: Record<string, any> = {};
+                const client: Record<string, any> = {};
+                for (const [key, options] of entries) {
+                    let current = options.context === 'server' ? server : client;
+                    const keys = key.split('__');
+                    for (let i = 0; i < keys.length - 1; i++) {
+                        const key = snakeCaseToCamelCase(keys[i]);
+                        current[key] ??= {};
+                        current = current[key];
+                    }
+
+                    var targetType: string = options.type;
```

Biome (1.9.4)

[error] Use let or const instead of var.

A variable declared with var is accessible in the whole body of the function. Thus, the variable can be accessed before its initialization and outside the block where it is declared.
See MDN web docs for more details.
Unsafe fix: Use 'let' instead.


(lint/style/noVar)